puppet/zulip/files/postgresql/process_fts_updates

```diff
@@ -27,7 +30,21 @@ logger.setLevel(logging.DEBUG)
 
 logger.info("process_fts_updates starting")
 
-conn = psycopg2.connect("user=zulip")
+sys.path.insert(0, '/home/zulip/deployments/current')
+os.environ['DJANGO_SETTINGS_MODULE'] = 'zproject.settings'
+import zproject.settings
+
+if settings.REMOTE_POSTGRES_HOST != '':
+    postgre_password = ''
+    if zproject.settings.get_secret("postgres_password") is not None:
+        postgre_password = "password='%s'" % zproject.settings.get_secret("postgres_password")
```

Comment:
Probably cleaner to access via settings.DATABASES['default']['PASSWORD'] rather than reading `get_secret`, so that `get_secret` can rename a settings.py-internal implementation detail.